wordpress-mobile/gutenberg-mobile
# Add git submodule dep updates via Dependabot

Experimental git-submodule dependency updates being done semi-automatically via Dependabot

Should be doable based on this older news post that Dependabot [supports git submodules updates](https://dependabot.com/blog/dependabot-now-supports-git-submodules/), plus the [related documentation](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem).

FWIW, here's the list of PRs that Dependabot has opened in this repo in the past: https://github.com/wordpress-mobile/gutenberg-mobile/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+is%3Aclosed

To test:
I think that we can't test something while this PR is unmerged. After we merge we can check whether Dependabot indeed opens new PRs to auto-update the git submodule references.

PR submission checklist:
